packages/core/src/pie-chart.ts

```diff
@@ -127,7 +127,7 @@ export class PieChart extends BaseChart {
 			.attr("height", `${diameter}px`);
 
 		this.innerWrap
-			.style("transform", `translate(${radius}px,${radius}px)`)
+			.attr("transform", `translate(${radius},${radius})`)
```

Comment:
What was the issue here?

Reply:
its just different than the other charts extending the basechart and changing it allows it to be more consistent with the other charts for spacing purposes. Returning an array with the translations will work effectively for all chart types now.

Reply:
ok